__tests__/materials/material.mutation.test.js

```diff
@@ -18,11 +18,11 @@ let token;
 let materialId;
 const materialDoesNotExistId = '1f2ad410eb01783384e6111b';
 
-describe('test material mutations', () => {
-  beforeAll(async () => {
-    token = await adminLogin(user);
-  });
-  test('#1 should add material to database', async () => {
+beforeAll(async () => {
```

Comment:
You still need root `describe` statement with some name like `Materials mutations tests`

Reply:
fixed